src/registry.ts

```diff
@@ -3,8 +3,12 @@ import * as ecs from "@rbxts/jecs";
 
 import { createSignal, type Signal } from "./signal";
 
-export interface Wildcard {}
-export interface ChildOf {}
+export interface Flag {
```

Comment:
Change the name of this interface and every subsequent dependents on this interface to use "Tag"

Reply:
Tag is already used for Entity<undefined>. Should I just get rid of that then?

Reply:
> Should I just get rid of that then?
Yeah, that sounds reasonable.